content/en/docs/instrumentation/js/getting-started/browser.md

```diff
@@ -3,7 +3,7 @@ title: Browser
 aliases: [/docs/js/getting_started/browser]
 ---
 
-This guide uses the example application in HTML & javascript provided below, but the steps to instrument your own application should be broadly the same.
+This guide uses the example application in HTML & typescript/javascript provided below, but the steps to instrument your own application should be broadly the same.
 
 ## Example Application
```

Comment:
Before this section we should have a section entitled prerequisites

> ## Prerequisites

You'll probably want to mention Node.js requirements.

In the edits presented in this file, there seems to be an assumption that `tsc` is installed and available. If that is the case, then it should be mentioned. Otherwise, (or maybe regardless), readers to be referred to https://www.typescriptlang.org/download if they need to install TypeScript.

I particularly like the NPM based install, but then that would require using `npx tsc` instead of `tsc` below. WDYT @svrnm?

Reply:
Is `npx` installed by default? I am not 100% sure anymore, but if so, yes, let's reduce the steps people have to take.

Reply:
Yes the `npx` gets installed by default with npm versions above 5 I guess